### PR TITLE
Improve using Qt flags by direct use enums.

### PR DESCRIPTION
**Pull request by Czaki**
# Description

In this PR I change the usage of Qt enums from `Qt.enum_item` to `Qt.Enum_Name.enum_item`. this reduces warnings in PyCharm and improves readability by additional context. 

Also, there are minor fixes by updating docstrings, and simplifying code. 

## Type of change

- [x] Refactor

# References

# How has this been tested?

- [ ] example: the test suite for my feature covers cases x, y, and z
- [ ] example: all tests pass with my change
- [ ] example: I check if my changes works with both PySide and PyQt backends
      as there are small differences between the two Qt bindings.  

## Final checklist:
- [ ] My PR is the minimum possible work for the desired functionality
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If I included new strings, I have used `trans.` to make them localizable.
      For more information see our [translations guide](https://napari.org/developers/translations.html).

### CODE REVIEW

**Review by tlambert03 — APPROVED** [review]
thanks, love this change.  Is there anything in here besides the enum scoping changes and the few minor syntax changes I saw that I should be looking out for?   If not, LGTM

File: napari/_qt/dialogs/qt_notification.py [hunk 1]
```diff
@@ -351,8 +359,6 @@ def from_notification(
 
         from ...utils.notifications import ErrorNotification
 
-        actions = notification.actions
-
```
Comment (Czaki) [inline napari/_qt/dialogs/qt_notification.py:355]: I think that these lines are obsolete, so I remove them, but maybe there are some side effects?

File: napari/_qt/layer_controls/qt_labels_controls.py [hunk 2]
```diff
@@ -163,7 +161,7 @@ def __init__(self, layer):
 
         # shuffle colormap button
         self.colormapUpdate = QtModePushButton(
-            None,
+            layer,
```
Comment (Czaki) [inline napari/_qt/layer_controls/qt_labels_controls.py:164]: maybe here, but I assume that it was a bug.

**Review by Czaki — COMMENTED** [review]
@tlambert03 The only two places that may need a special look, but I'm not sure.

File: napari/_qt/layer_controls/qt_labels_controls.py (hunk 2 above)
Comment (tlambert03) [reply napari/_qt/layer_controls/qt_labels_controls.py:164]: also looks good

**Review by tlambert03 — COMMENTED** [review]
Will leave this for about 24 hours in case anyone wants to look, but since this one is prone to merge conflicts, we shouldn't leave it around too long.

**Pull request merged**